db/migrate/20210323204755_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :password_digest
```

Comment:
We should make sure to add in the timestamps here. Easy fix and I can walk you through that.